src/helpers/utils.js

```diff
@@ -36,3 +38,8 @@ export function getTimeDifferenceInDaysFromToday(dateToCheck) {
   const currentDate = new Date();
   return getTimeDifferenceInDays(dateToCheck, currentDate);
 }
+
+export const onLogout = async (dispatch) => {
+  localStorage.clear();
+  await dispatch(logout());
+};
```

Comment:
Could be used in LogoutButton.js there is a similar function.

Reply:
There is some difference with redirecting after logout, but nevertheless, it could be merged and kept consistent.

Reply:
done